plugins/aap-backend/src/providers/AapResourceEntityProvider.ts

```diff
@@ -114,9 +117,23 @@ export class AapResourceEntityProvider implements EntityProvider {
     const jobTemplates = await listJobTemplates(
       this.baseUrl,
       this.authorization,
-    );
+    ).catch(err => {
+      console.error('Failed to fetch AAP job templates', err);
+    });
+
+    const workflowJobTemplates = await listWorkflowJobTemplates(
+      this.baseUrl,
+      this.authorization,
+    ).catch(err => {
+      console.error('Failed to fetch AAP workflow job templates', err);
+    });
+
+    const templates = [
+      ...(jobTemplates || []),
+      ...(workflowJobTemplates || []),
+    ];
```

Comment:
Can we fetch them in parallel instead of in sequential order? There's no point of async functions if we `await` each call in sequence 🙂 

Something like `Promise.all` should do.

Reply:
`Promise.all` is rejected if any of the elements are rejected but here my intent is even if the promise for JobTemplates fails but if WorkflowJobTemplate passes then the catalog should show the entities for that 🤔

Reply:
Thanks @tumido, have updated it and made use of `Promise.allSettled`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled